Directory.Packages.props

```diff
@@ -13,6 +13,11 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- System packages -->
+    <PackageVersion Include="Microsoft.EntityFrameworkCore" Version="7.0.2" />
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.2" />
+    <PackageVersion Include="Polly" Version="7.2.3" />
```

Comment:
Eeeh, I think I might wanna remove Polly and use the Orleans built-in retry helpers.